nuxt-pinia-tailwind/components/UserProfile/UserProfile.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div>
+    <div class="w-[260px] h-[260px]">
+      <img
+        :src="userProfile.avatar"
+        alt="Avatar"
+        width="260"
+        height="260"
+        class="absolute rounded-full shadow z-30"
+      />
```

Comment:
I was having an error here in my IDE, following [this](https://github.com/nuxt/typescript/issues/537#issuecomment-1050054649) helped me solve the issue. Just installing the package was enough for me, I didn't have to change the tsconfig file

![image](https://user-images.githubusercontent.com/39612839/193307181-f45afc1e-17ba-4782-a494-00f79530af97.png)